airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -112,6 +112,10 @@ class PodReattachFailure(AirflowException):
     """When we expect to be able to find a pod but cannot."""
 
 
```

Comment:
I wonder if we should have exceptions.py like we have in other providers

Reply:
I like the idea and can do it. Should it be part of this PR though?

Reply:
Yes so we don't need to deprecate in the future.
You can leave the existed exception as is. We can migratre it in another PR

Reply:
Created _exceptions.py_ and moved  `IdenticalLabelPodError` there. `PodReattachFailure` is left as is. Please verify.

Reply:
We only have one example of `test_exceptions.py.py` in _tests/providers/docker/test_exceptions.py_. I cannot follow its pattern with this operator.

Since I added `IdenticalLabelPodError` exception only to replace `PodReattachFailure` which did not seem  to be used anymore, and then it turned out that I can't just replace it because of the deprecation policy, I don't think it is that important for me to keep it. 

I removed `IdenticalLabelPodError` and put `AirflowException` back in its place.